docs/architecture/microservices/microservice-ddd-cqrs-patterns/domain-events-design-implementation.md

```diff
@@ -249,7 +249,7 @@ Another way to map events to multiple event handlers is by using types registrat
 
 You can build all the plumbing and artifacts to implement that approach by yourself. However, you can also use available libraries like [MediatR](https://github.com/jbogard/MediatR) that uses your IoC container under the covers. You can therefore directly use the predefined interfaces and the mediator object's publish/dispatch methods.
 
-In code, you first need to register the event handler types in your IoC container, as shown in the following example at [eShopOnContainers Ordering microservice](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/Services/Ordering/Ordering.API/Infrastructure/AutofacModules/MediatorModule.cs):
+In code, you first need to register the event handler types in your IoC container, as shown in the following example at [eShopOnContainers Ordering microservice](https://github.com/dotnet-architecture/eShopOnContainers/blob/2c94c382696aeab97cb28743a1637ab8d31d494a/src/Services/Ordering/Ordering.API/Infrastructure/AutofacModules/MediatorModule.cs):
```

Comment:
@IEvangelist It looks like Autofac was completely removed from the sample as part of some modernization work. Not sure if you want to update this article.